app/src/main/java/com/example/workoutplaner/Navigation.java

```diff
@@ -0,0 +1,14 @@
+package com.example.workoutplaner;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+
+public class Navigation extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_navigation2);
```

Comment:
Why is it navigation2? Do we have another navigation?

Reply:
Somehow i missed, there is only 1 navigation, i fixed the naming of it.

Reply:
should the `R.layout.activity_navigation2` also have 2 in the name?

Reply:
It should be just navigatio, without 2.